LICENSE.txt

```diff
@@ -186,7 +186,7 @@ Apache License
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright 2014-2015 Spotify AB
+   Copyright 2014-2018 Spotify AB
```

Comment:
This actually shouldn't include our copyright; it's meant to be a template and should be copied verbatim from [the Apache license](https://www.apache.org/licenses/LICENSE-2.0.txt).